### PR TITLE
Correct logger messages in plotting.py

### PR DESCRIPTION
**Pull request by vinferrer**
Closes nothing

Changes proposed in this pull request:

- the logger messages in plotting.py were giving errors, now they work
-

### CODE REVIEW

**Review by eurunuela — APPROVED** [review]
LGTM!

**Comment by eurunuela** [comment]
Fix the conflicts and merge!

File: aroma/plotting.py [hunk 1]
```diff
@@ -53,8 +53,8 @@ def classification_plot(myinput, out_dir):
     # get counts
     ncomp = len(df)
     nmot = len(df.loc[df[0] == "True"])
-    LGR.info('Found', nmot, 'head motion-related components in a total of',
-             ncomp, 'components.')
+    LGR.info(f'Found {nmot} head motion-related components in a total of',
```
Comment (tsalo) [inline aroma/plotting.py]: Needs a space at the end.

Comment (tsalo) [reply aroma/plotting.py]: Also, f-strings limit us to 3.6+, I believe. I never planned to support Python 2, but I haven't thought about 3.5.

Comment (tsalo) [reply aroma/plotting.py]: Ah, it looks like we have `setup.cfg` limited to 3.6+, so it's all good. We just need the space at the end of the first line.

**Comment by vinferrer** [comment]
This is ready for merging i Think

File: aroma/plotting.py [hunk 2]
```diff
@@ -23,54 +23,40 @@ def classification_plot(in_file, out_dir):
...
-                       float(vals[5])])
-
-        if count == 0:
-            df = pd.DataFrame.from_records(res)
-        else:
-            df2 = pd.DataFrame.from_records(res)
-            df = df.append(df2, ignore_index=True)
-
-        count += 1
+    assert isinstance(in_file, str)
```
Comment (eurunuela) [inline aroma/plotting.py]: A nicer way of doing this would be to use the `os` package and an if statement checking that the path exists.

**Review by eurunuela — COMMENTED** [review]
I don't understand what you mean by the title of this PR. 

I think you're making more changes than just making the logger work, and this should be mentioned in the comment and the title of the PR.

File: aroma/plotting.py (hunk 2 above)
Comment (vinferrer) [reply aroma/plotting.py]: This change comes from merging with main I didn't add this line of code

Comment (eurunuela) [reply aroma/plotting.py]: I would change it then.

@tsalo any reason why we have an assert here?

Comment (vinferrer) [reply aroma/plotting.py]: or maybe i didn't accept the changes when mergin with main. I can't remember now

Comment (eurunuela) [reply aroma/plotting.py]: Well, let's see what @tsalo has to say. I guess he added that line.

Comment (tsalo) [reply aroma/plotting.py]: Sorry for the delay on this. I added the assert to make sure that we didn't accept lists of files anymore, I believe.

**Comment by vinferrer** [comment]
Can we merge this then?

**Pull request merged**